src/Tribe/Support.php

```diff
@@ -44,10 +44,14 @@ private function __construct() {
 			$this->must_escape = (array) apply_filters( 'tribe_help_must_escape_fields', $this->must_escape );
 			add_action( 'tribe_help_pre_get_sections', array( $this, 'append_system_info' ), 10 );
 			add_action( 'delete_option_rewrite_rules', array( $this, 'log_rewrite_rule_purge' ) );
+
+			add_action( 'rest_api_init', array( $this, 'create_sysinfo_endpoint' ) );
```

Comment:
`create_sysinfo_endpoint()` is a static method - we should switch to `array( __CLASS__, 'create_sysinfo_endpoint' )` or `'Tribe__Support::create_sysinfo_endpoint'` or similar.